src/Page.ts

```diff
@@ -919,6 +919,29 @@ export class Page extends EventEmitter {
     }
   }
 
+  async emulateVisionDeficiency(type?: string): Promise<void> {
+    const visionDeficiencies = new Set([
+      'none',
+      'achromatopsia',
+      'blurredVision',
+      'deuteranopia',
+      'protanopia',
+      'tritanopia',
+    ]);
+    try {
+      assert(
+        !type || visionDeficiencies.has(type),
+        `Unsupported vision deficiency: ${type}`
+      );
+      await this._client.send('Emulation.setEmulatedVisionDeficiency', {
+        // @ts-ignore: What's the proper way of dealing with this? */
```

Comment:
@jackfranklin Any ideas here?

```
src/Page.ts:938:9 - error TS2322: Type 'string' is not assignable to type '"none" | "achromatopsia" | "blurredVision" | "deuteranopia" | "protanopia" | "tritanopia"'.

938         type: type || 'none',
            ~~~~

  src/protocol.d.ts:4661:13
    4661             type: "none"|"achromatopsia"|"blurredVision"|"deuteranopia"|"protanopia"|"tritanopia";
                     ~~~~
    The expected type comes from property 'type' which is declared here on type 'setEmulatedVisionDeficiencyParameters'
```

Reply:
I think in the function parameter you don't want to use `type?: string` but `type?: "none" | "achromatopsia" | ...`

Reply:
Gotcha, thanks!